src/hpm.h

```diff
@@ -6,24 +6,44 @@ extern "C" {
 #endif
 
     #include <stdbool.h>
+    #include <stdint.h>
 
-    typedef int (*hpm_enviaSerie_t)(const char *carga,char largo);
+    typedef struct {
+        int16_t pm10;
+        int16_t pm25;
+    } HPMdata_t;
+    
+     bool NCupdated(void);//Copiado de nmeacayenne.h
```

Comment:
Esta funcion les está causando problemas, hay que renombrarla
```suggestion
    bool HPMupdated(void);
```